dxvk.conf

```diff
@@ -339,6 +339,24 @@
 # environment variable is not set, otherwise it will be
 # ignored. The syntax is identical.
```

Comment:
> The syntax is identical.

This still hints you to look at the readme, so i'd just note here that the values are comma separated.

Reply:
I just wanted to avoid opening the config file and the github readme
I'm lazy